src/Pact/Analyze/Check.hs

```diff
@@ -1098,7 +1106,7 @@ getFunChecks env@(CheckEnv tables consts propDefs moduleData _cs _g) refs = do
       _            -> error
         "invariant violation: anything but a TopFun is unexpected in funChecks"
 
-  pure (funChecks'', invariantChecks)
+  pure (nub . concat <$> funChecks'', invariantChecks)
```

Comment:
same here. Do you mind describing the problem this `nub` is solving?

Reply:
See above